pkg/config/settings/runtime_setting_block_profile_rate.go

```diff
@@ -0,0 +1,42 @@
+package settings
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/profiling"
+)
+
+// RuntimeBlockProfileRate wraps runtime.SetBlockProfileRate setting
+type RuntimeBlockProfileRate (string)
+
+// Name returns the name of the runtime setting
+func (r RuntimeBlockProfileRate) Name() string {
+	return string(r)
+}
+
+// Description returns the runtime setting's description
+func (r RuntimeBlockProfileRate) Description() string {
+	return "This setting controls the fraction of goroutine blocking events that are reported in the blocking profile"
```

Comment:
Let's mention explicitly that this only affects _internal_  profiles (not application profiles that the trace-agent forwards to DD)

```suggestion
	return "This setting controls the fraction of goroutine blocking events that are reported in the internal blocking profile"
```